app/views/feedback/create.html.haml

```diff
@@ -0,0 +1,3 @@
+Thanks for your feedback
+
+= link_to "Return to gov.au", @feedback.url
```

Comment:
This could possibly be a redirect with a flash message rather than a whole page just to say "Thanks for your feedback." Just a thought though.